src/Components/TodoApp/TodoApp.tsx

```diff
@@ -0,0 +1,55 @@
+import {
+  useState,
+  useContext,
+  useRef,
+  useEffect,
+} from 'react';
+import './todoApp.css';
+import { DispatchContext } from '../Store';
+
+export const TodoApp = () => {
+  const dispatch = useContext(DispatchContext);
+  const [title, setTitle] = useState('');
+  const titleField = useRef<HTMLInputElement>(null);
+
+  useEffect(() => {
+    if (titleField.current) {
+      titleField.current.focus();
+    }
+  }, []);
+
+  const handleSubmit = (event: React.FormEvent) => {
+    event.preventDefault();
+    const newTitle = title.trim();
+
+    if (!newTitle) {
+      setTitle('');
+
+      return;
+    }
+
+    dispatch({ type: 'add', payLoad: newTitle });
```

Comment:
`payLoad` property in dispatch function should be spelled as `payload`. It's a common naming convention in Redux to use 'payload' for the dispatched data.